src/ResourceManager/Compute/Commands.Compute/Extension/Diagnostics/StartAzureRmVmssDiagnosticsStreaming.cs

```diff
@@ -0,0 +1,116 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Compute.Common;
+using Microsoft.Azure.Commands.Resources.Models;
+using Microsoft.Azure.Management.Compute;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.VisualStudio.EtwListener.Common;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Management.Automation;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Compute.Extension.Diagnostics
+{
+    [Cmdlet(
+        VerbsLifecycle.Start,
+        ProfileNouns.VirtualMachineScaleSetDiagnosticsStreaming,
+        SupportsShouldProcess = true)]
+    [OutputType(typeof(PSEtwEvent))]
+    public class StartAzureRmVmssDiagnosticsStreaming : EtwStreamingVmssCmdletBase
+    {
+        [Parameter(
+           Mandatory = true,
+           Position = 0,
+           ValueFromPipelineByPropertyName = true,
+           HelpMessage = "The resource group name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Alias("ResourceName")]
```

Comment:
Add "Name" as an alias.  VirtualMachineScaleSet object has a property "Name".  So in order for this property to get the value from the pipeline, the parameter should be either "Name" or has "Name" as an alias